RTB/Piano_Progetto.tex

```diff
@@ -142,15 +142,98 @@ \subsubsection{Informativi}
+        \item \textbf{Monitoraggio e Revisione dei Rischi}:
+        Questa fase consiste in un monitoraggio continuo dei rischi identificati, per garantire l'efficacia delle misure adottate e per rilevare tempestivamente la comparsa di nuovi rischi;
+    \end{enumerate}
+    I rischi identificati sono stati suddivisi in tre categorie principali:
+    \begin{itemize}
+        \item \textbf{Rischi Tecnologici}: Riguardano le incertezze e i possibili fallimenti legati alle tecnologie impiegate nel progetto, come malfunzionamenti o difficoltà nell'integrazione; 
+        \item \textbf{Rischi di Comunicazione}: Riguardano le difficoltà nello scambio di informazioni tra i membri del gruppo o con la proponente, che potrebbero compromettere il corretto svolgimento del progetto;
+        \item \textbf{Rischi di Pianificazione}: Riguardano le problematiche relative alla gestione del tempo, delle scadenze e delle risorse, inclusi eventuali ritardi, stime imprecise o spreco delle risorse;
+    \end{itemize}
+
+    \subsubsection{Struttura dei Rischi}
+    Il formato utilizzato per la classificazione dei rischi è R[Tipo][Indice], dove:
+    \begin{itemize}
+        \item \textbf{R}: abbreviazione di "Rischio";
+        \item \textbf{Tipo}: rappresenta la categoria del rischio e può essere:
+        \begin{itemize}
+            \item \textbf{T}: Rischio Tecnologico;
+            \item \textbf{C}: Rischio di Comunicazione;
+            \item \textbf{P}: Rischio di Pianificazione;
+        \end{itemize}
+        \item \textbf{Indice}: numero progressivo che identifica univocamente ciascun rischio all'interno della categoria di appartenenza.
+    \end{itemize}
+
+\subsection{Rischi Tecnologici}
+    \subsubsection{RT1 - Complessità delle nuove tecnologie}
+    \begin{itemize}
+        \item \textbf{Descrizione}: Il progetto richiede l'utilizzo di tecnologie poco conosciute o con cui il gruppo ha un'esperienza limitata. Di conseguenza l'apprendimento e l'adattamento a queste nuove tecnologie richiede del tempo, che potrebbe comportare rallentamenti nel progresso del progetto;
+        \item \textbf{Gravità}: Alta;
+        \item \textbf{Occorrenza}: Alta;
+        \item \textbf{Possibili Attenuazioni}: Per affrontare la complessità delle nuove tecnologie, verranno adottate diverse strategie. L'azienda proponente si è resa disponibile ad organizzare incontri focalizzati su tecnologie specifiche, per risolvere eventuali dubbi. Inoltre, i membri del gruppo collaboreranno attivamente, condividendo le proprie informazioni al fine di raggiungere un livello condiviso di conoscenze. Infine, verrà dedicato del tempo alla fase di apprendimento, consentendo al gruppo di esplorare diverse tecnologie, identificare i punti di forza e le criticità, così da accelerare l'apprendimento e prevenire errori futuri;
```

Comment:
Provare a dividere questa sezione  del rischio in punti elenco più che fare un discorso unico, per il resto non trovo errori o cambiamenti da apportare